noir-projects/noir-contracts/contracts/docs_example_contract/src/options.nr

```diff
@@ -1,18 +1,21 @@
-use crate::types::card_note::{CARD_NOTE_LEN, CardNote};
+use crate::types::card_note::CardNote;
```

Comment:
Sneaked changes in this file in (Nico 😝) - we relied on VALUE_NOTE_LEN being exported which is an anti-pattern.